eslintManager.js

```diff
@@ -152,6 +159,16 @@ function getRuleDetails(ruleName) {
         });
 }
 
+function getPluginName(scope, plugin) {
+    return [scope, plugin].filter(x => !!x).join('/');
```

Comment:
Please change this to the following:

```suggestion
    return [scope, plugin].filter(x => x !== undefined).join('/');
```